packages/postgres/migrations/1752685843945_migrate-old-users.js

```diff
@@ -0,0 +1,102 @@
+const matrixUserIds = [
+  '@0xpranayy:boxel.ai',
+  '@agent:boxel.ai',
+  '@aibot:boxel.ai',
+  '@andrecastro89:boxel.ai',
+  '@athompson19:boxel.ai',
+  '@b9:boxel.ai',
+  '@banksia:boxel.ai',
+  '@breakdance12:boxel.ai',
+  '@btc:boxel.ai',
+  '@buck.doyle+202410231246:boxel.ai',
+  '@bucktest:boxel.ai',
+  '@burcu:boxel.ai',
+  '@burcunoyan1:boxel.ai',
+  '@burcunoyan:boxel.ai',
+  '@cgardella:boxel.ai',
+  '@chad924:boxel.ai',
+  '@chris:boxel.ai',
+  '@chuanstack:boxel.ai',
+  '@david:boxel.ai',
+  '@dezertfox:boxel.ai',
+  '@dick:boxel.ai',
+  '@dmartinez21:boxel.ai',
+  '@dml:boxel.ai',
+  '@doug:boxel.ai',
+  '@doug_funny:boxel.ai',
+  '@dtaylor56:boxel.ai',
+  '@fadhlanr:boxel.ai',
+  '@harry:boxel.ai',
+  '@hassan1:boxel.ai',
+  '@hassan2:boxel.ai',
+  '@hassan3:boxel.ai',
+  '@hassan4:boxel.ai',
+  '@ian:boxel.ai',
+  '@iantest2:boxel.ai',
+  '@iantest3:boxel.ai',
+  '@iantest4:boxel.ai',
+  '@iantest:boxel.ai',
+  '@itranslate:boxel.ai',
+  '@jacthong:boxel.ai',
+  '@janderson23:boxel.ai',
+  '@jbrown44:boxel.ai',
+  '@jhall77:boxel.ai',
+  '@jtjr:boxel.ai',
+  '@jurgen1:boxel.ai',
+  '@jurgen5:boxel.ai',
+  '@jurgen:boxel.ai',
+  '@jurgenwerk2:boxel.ai',
+  '@jurgenwerk:boxel.ai',
```

Comment:
This is one of my usernames and actually there is a user table entry for this username

Reply:
Ah sorry I see now, it's just a list of all prod users. All good

Reply:
yeah, i literally jsut got a csv of all the prod matrix usernames and added them. the script just skips over any ones that already are in the users table.